PyInstaller/hooks/hook-PyQt5.QtQml.py

```diff
@@ -36,3 +38,481 @@
+                            if self.is_raw_string(second):
+                                return True
+                            else:
+                                return False
+                            break
+
+                        else:
+                            continue
+
+    def is_raw_string(self, raw):
+
+        # Here we read the line to see if there is
+        # a genuine qml filepath.
+        if re.findall('([+]|,|[)])', raw) or not re.findall('(\'|")', raw):
+            # It is a variable
+            return False
+
+        else:
+            if re.findall('qrc:', raw):  # it is a qrc
+                return False
+            else:
+                if raw[0] == '"':
+                    self.main_qml = raw.replace('"', '')
+                    return True
+
+                elif raw[0] == "'":
+                    self.main_qml = raw.replace("'", "")
+                    return True
+                else:
+                    return False
+
+    def reconstruct_qml_path(self):
+
+        # This is where we verify if the
+        # qml filepath is a relative path
+        # to the application's folder
+        # if it's not, we simply do nothing.
+        # it can't be a full path.
+        if os.path.exists(self.main_qml):
+            return False
+        else:
+            self.append_to_data(self.analysis_folder, self.main_qml)
+            self._first_handle(self.analysis_folder, self.main_qml)
+
+    def _crawl_qml_file(self, c_file):
+
+        # This is where we parse the qml file to look for
+        # import statements.
+
+        with open(c_file, mode='r', encoding='utf-8') as fh:
+            for line in fh:
+                if re.findall('[ ]+[//]', line):
+                    # skip it's a comment line
+                    continue
+                else:
+                    self.find_imports(line)
+                    # set the qml file to the current qml file
+                    # since the image path should be relative to
+                    # that file
+                    self.current_qml_file = c_file
+                    self.find_images(line)
+
+    def find_imports(self, statement):
+        import_stats = re.findall('import .*?.*?.*?$', statement)
+        if import_stats:
+            # since we are parsing this line by line
+            # there will always be just one entry in import stats
+            self._sanitize_imports(import_stats[0])
+
+    def _sanitize_imports(self, stat):
+
+        # Remove the (') apostrophies that comes
+        # which are found in them
+        if len(stat) > 8:
+            clean = stat[7:-1]
+            if clean not in self.raw_import_stats:
+                self.raw_import_stats.append(clean)
+        else:
+            # Unkwown import type
+            return False
+
+    def find_images(self, statement):
+        for keyword in self.image_search_words:
+            img_query = re.findall('["|\'].*?.*?.*?[.]' + keyword + '["|\']',
```

Comment:
Can you please give an example, what exacly should be matched.

I assume this could be simplified a lot. E.g one could use egex-grouping to remove the quotes.

Also this includes some errors in the regexp, see comment for line 363.

Additionally I assume the opening quote should be the same as the closing (`"lalal'` should not be matched), thus a regex-backreference is required.